static/app/components/interactionStateLayer.tsx

```diff
@@ -71,7 +71,8 @@ const InteractionStateLayer = styled(
         `
       : ''}
 
-  *:disabled > && {
```

Comment:
Did this even work? Iirc `> &&` is not a valid selector?

Reply:
It did! The problem was it only matched `InteractionStateLayer`s whose direct parent (`>`) is disabled, and not any ancestor further up the line.